cfy_manager/main.py

```diff
@@ -871,33 +921,35 @@ def remove(verbose=False, force=False, config_file=None):
         logger.warning('--force is deprecated, does nothing, and will be '
                        'removed in a future version')
 
-    logger.notice('Removing Cloudify Manager...')
+    removed_services = [service_name.split('_')[0].capitalize() for
+                        service_name in get_installed_services_names()]
+
+    logger.notice('Removing Cloudify %s...', (
+        'Manager' if is_all_in_one_manager() else ', '.join(removed_services)))
 
     should_stop = _are_components_configured()
-    components = _get_components()
-    for component in reversed(components):
+    components_to_remove = _get_components(
```

Comment:
As you'll otherwise be reversing this repeatedly later, maybe just reverse it here?

Reply:
Right! Will change it.